query/sql/SqlBase.g4

```diff
@@ -0,0 +1,592 @@
+/*
```

Comment:
need to check with Ze/Jian on how the license works here

Reply:
got OK from OSS legal